src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -29,7 +29,7 @@
       <NoWarn>NU1701</NoWarn>
     </PackageReference>
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.5350001-beta-fc119b98" />
-    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.1.1-beta03" />
+    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.1.1-beta03-10111" />
```

Comment:
Isn't this version number messed up? I.e., version 1.1.1-beta03 takes precedence over any prerelease versions. I think this needs to go to beta4?

Reply:
having "beta" is still considered prerelease. This version did show up in the updates in Nuget Package manager